web/modules/custom/nica_custom/templates/nica-custom-cv.html.twig

```diff
@@ -0,0 +1,29 @@
+{# nica_custom/templates/nica_custom_resume.html.twig #}
+
+<div class="container">
```

Comment:
@davidsolorzano20, copy this theme/twig file into the theme too. I think this is very close.